FlowCrypt.xcodeproj/project.pbxproj

```diff
@@ -831,7 +831,7 @@
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
 				DEBUG_INFORMATION_FORMAT = dwarf;
-				ENABLE_BITCODE = YES;
+				ENABLE_BITCODE = NO;
```

Comment:
What is the reason to disable bitcode in development?

Reply:
I changed it, because I can not run tests. I have an error that says bitcode and bundle could not be used together.

Reply:
Interesting. I will revert this change for now (bitcode is a good thing to have for debugging), and later we can take a look at what causes the issues with tests, in another issue.